SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you've discovered a security issue, please report it through the form linked below, which will create a secure, private ticket. 
+https://bugzilla.mozilla.org/form.web.bounty
+
+MDN is also eligible for [Mozilla's Security Bug Bounty Program](https://www.mozilla.org/en-US/security/bug-bounty/).
```

Comment:
using the link https://www.mozilla.org/en-US/security/web-bug-bounty/ is more accurate